docs/getting-started/first-abn.md

```diff
@@ -83,17 +79,24 @@ In another shell, run a script to generate load from multiple users:
 
 ## Deploy candidate
 
-Deploy the candidate version of the *backend* component, naming it `backend-candidate-1`.
+Release the application adding a second (candidate) version named `backend-candidate-1`:
 
 ```shell
-kubectl create deployment backend-candidate-1 --image=iter8/abn-sample-backend:0.17-v2
-kubectl label deployment backend-candidate-1 iter8.tools/watch="true"
-
-kubectl expose deployment backend-candidate-1 --name=backend-candidate-1 --port=8091
+cat <<EOF | helm upgrade --install backend $CHARTS/release -f -
+environment: deployment
+application: 
+  port: 8091
+  versions:
+  - metadata:
+      name: backend
+    image: iter8/abn-sample-backend:0.17-v1
+  - metadata:
+      name: backend-candidate-1
+    image: iter8/abn-sample-backend:0.17-v2
+EOF
 ```
 
-Until the candidate version is ready; that is, until all expected resources are deployed and available, calls to `Lookup()` will return only the version number `0`; the existing version.
-Once the candidate version is ready, `Lookup()` will return both version numbers (`0` and `1`) so that requests can be distributed across versions.
+While the candidate version is deploying, `Lookup()` will continue to return just the primary version. Once the candidate is ready, it will begin recommending both versions allowing traffic to be distributed across both.z
```

Comment:
```suggestion
While the candidate version is deploying, `Lookup()` will only return the primary version. Once the candidate is ready, it will begin recommending both versions, allowing traffic to be distributed across both.
```

Reply:
I think it should be mentioned here that the `versions` list is ordered and the first version is the primary version.